### PR TITLE
docs: Fixes for SentrySDK and Hub

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

Minor fixes to code docs of SentrySDK and Hub.

## :bulb: Motivation and Context

We want to have proper code docs for our public classes.

## :green_heart: How did you test it?
CI.

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I updated the CHANGELOG if needed
- [x] I updated the docs if needed
- [x] Review from the native team if needed
- [x] No breaking changes

### CODE REVIEW

File: Sources/Sentry/Public/SentryHub.h [hunk 1]
```diff
@@ -196,13 +202,15 @@ SENTRY_NO_INIT
...
- * @return BOOL If instance of `integrationClass` exists within
- * `SentryHub.installedIntegrations`.
+ *
+ * @return BOOL If instance of `integrationClass` exists within `SentryHub.installedIntegrations`.
  */
 - (BOOL)isIntegrationInstalled:(Class)integrationClass;
 
 /**
- * Set global user -> thus will be sent with every event
+ * Set user to the Scope of the Hub. Thus, the Hub is going to send them with every event.
```
Comment (marandaneto) [inline Sources/Sentry/Public/SentryHub.h:211]:
```suggestion
 * Set user to the Scope of the Hub.
```

this is pretty much the same for all the Scope fields, eg addBreadcrumb, and we don't say that, I'd just skip it as its the intended behaviour anyway

File: Sources/Sentry/Public/SentrySDK.h [hunk 2]
```diff
@@ -216,7 +219,10 @@ SENTRY_NO_INIT
 @property (nonatomic, class, readonly) BOOL crashedLastRun;
 
 /**
- * Set global user -> thus will be sent with every event
+ * Set user to the current Scope of the current Hub. Thus, the SDK is going to send them with every
+ * event.
```
Comment (marandaneto) [inline Sources/Sentry/Public/SentrySDK.h:223]:
```suggestion
 * Set user to the current Scope of the current Hub.
```

File: Sources/Sentry/SentryHub.m [hunk 3]
```diff
@@ -401,9 +401,6 @@ - (id _Nullable)getIntegration:(NSString *)integrationName
     return [integrations objectAtIndex:[integrations indexOfObject:integrationName]];
 }
 
-/**
- * Set global user -> thus will be sent with every event
- */
```
Comment (marandaneto) [inline Sources/Sentry/SentryHub.m:406]: just to confirm, we are removing from here because the public ones are under Sentry/Public right?

**Review by marandaneto — APPROVED** [review]

File: Sources/Sentry/SentryHub.m (hunk 3 above)
Comment (philipphofmann) [reply Sources/Sentry/SentryHub.m:406]: Yes, no need to keep one in the implementation file.

**Pull request merged**